src/test/java/spoon/test/comment/testclasses/WildComments.java

```diff
@@ -115,5 +115,24 @@
 			/**** starts and ends with 4 * ****/
 			"** starts and ends with 4 * ***"
 	};
+
+	/* should not cause "Unexpected next line after last line" exception */
+	public void testSlash() {
+		/*/
+		slash and comment
+		*/
+
+		/*
+		comment and slash
+		/*/
+
+		/*/
+		slash and comment and slash
+		/*/
+
+		//*/
+
+		/*/*/
```

Comment:
Could you please move all these new comment examples above (to line >116). This test expects pairs of tested comment example and expected string content of the comment.

Reply:
Should I create a new file for them?

Reply:
> new file

no, just move them up and add expected content of the comment text. For example:
convert this
```java
/*/
slash and comment
*/
```
to
```java
/*/
slash and comment
*/
"/\nslash and comment"
```